sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
-## 1.0.0-preview.1 (Unreleased)
+## 4.0.0-beta.1 (Unreleased)
 
-- Produced the generated code for the administrative Key Vault tasks. It includes the backup and restore operations for whole Key Vault instances, and the role-based access control (RBAC) operations.
+The @azure/keyvault-admin package provides two clients, `KeyVaultAccessControlClient` and `KeyVaultBackupClient`.
+
+- The `KeyVaultAccessControlClient` allows assigning, deleting and retrieving role assignments, and retrieving role definitions.
```

Comment:
do you want to mention also more broadly that this handles role-based access control (aka RBAC)? Think it might be good to include the RBAC key word

Reply:
Oh! Can do.